recaf-ui/src/main/java/me/coley/recaf/ui/context/NumberLiteralContextBuilder.java

```diff
@@ -0,0 +1,73 @@
+package me.coley.recaf.ui.context;
+
+import com.github.javaparser.Position;
+import com.github.javaparser.ast.expr.EnclosedExpr;
+import com.github.javaparser.ast.expr.Expression;
+import com.github.javaparser.ast.expr.LiteralExpr;
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.MenuItem;
+import me.coley.recaf.ui.control.code.java.JavaArea;
+import me.coley.recaf.ui.util.Icons;
+import me.coley.recaf.util.EscapeUtil;
+import me.coley.recaf.workspace.resource.Resource;
+import org.fxmisc.richtext.model.TwoDimensional;
+
+import static me.coley.recaf.ui.util.Menus.action;
+
+public class NumberLiteralContextBuilder extends ContextBuilder {
+
+	private final Number value;
+	private final Expression expression;
+	private final JavaArea area;
+	private final int position;
+
+	public NumberLiteralContextBuilder(Number value, Expression expression, JavaArea area, int position) {
+		this.value = value;
+		this.expression = expression;
+		this.area = area;
+		this.position = position;
+	}
+
+	@Override
+	public ContextMenu build() {
+		ContextMenu menu = new ContextMenu();
+		MenuItem header = new MenuItem(EscapeUtil.escape(isLiteral(expression) ? "Number Literal" : "Number Expression"));
```

Comment:
No need for escape since the title is just going to be `Number Literal/Expression`

Reply:
Ok, now what about localization?

Reply:
`Lang.getBinding("your.key.here")` and add it to the `en_us` language file.